workspaces/ui-v2/Taskfile.yml

```diff
@@ -47,19 +48,13 @@ tasks:
       - >
         BUILD_PATH=build-cloud-spec-viewer
         yarn build:cloud
-    sources:
```

Comment:
I wasn't sure what the utility or correctness of these were in this context

Reply:
@notnmeyer does this break things? I have a very vague memory of you telling me that this `sources`/`generates` fields were important, but I can't quite remember why.

Reply:
Ah you're on vacation. No worries then, it can't break too many things, and if it does then we'll fix 'em.